### PR TITLE
adding search function and map

### PR DESCRIPTION
**Pull request by sophiecs333**
Could you please kindly review our hard work? :D

### CODE REVIEW

File: app/assets/stylesheets/config/_bootstrap_variables.scss [hunk 1]
```diff
@@ -23,3 +23,7 @@ $border-radius-lg: 2px;
 $border-radius-sm: 2px;
 
 // Override other variables below!
+.container {
+  padding-right: 0px!important;
+  padding-left: 0px!important;
+}
```
Comment (dumboluzz) [inline app/assets/stylesheets/config/_bootstrap_variables.scss:29]: what is this for?

**Review by finnstue — APPROVED** [review]
looks good!

File: app/assets/stylesheets/config/_bootstrap_variables.scss (hunk 1 above)
Comment (finnstue) [reply app/assets/stylesheets/config/_bootstrap_variables.scss:29]: to overwrite some padding that bootstrap adds to the container

File: db/seeds.rb [hunk 2]
```diff
@@ -35,12 +35,38 @@
...
+a.nick_name = "Jojo"
+a.age = 40
+a.price_per_day = 56
+a.height = 89
+a.weight = 78
+a.color = "gray"
+a.wool_type = "suri"
+a.address = "3103 Broadway, New York, NY 10027, USA"
+a.owner = User.order(Arel.sql('RANDOM()')).first
+a.save
```
Comment (dumboluzz) [inline db/seeds.rb:68]: seed the address for all the alpacas. eg.: Faker::Address.city and not only for 2

**Review by dumboluzz — DISMISSED** [review]

**Review by finnstue — APPROVED** [review]

**Pull request merged**